### ISSUE TITLE
enable overriding check arguments via ruleset config

### ISSUE THREAD

**Issue by TomasTomecek** [user-cont/colin#96]
Let's say we a have a [`MaintainerCheck`](https://github.com/user-cont/colin/blob/master/colin/checks/labels/maintainer.py). It would make things easier if we were able to create a ruleset where we would be able to override arguments, such as `reference_url` inside this ruleset. This will help us defining internal ruleset pointing to internal resources.

I fear that we might need to restructure the config quite a bit in order to enable this. Or maybe not, how about something like:
```
{
  "labels": {
    "required": [
      ...
        ]
      }
    ],
    "optional": [
      ...
    ],
    "extra-args": {
      "check-name": {
        "foo": "bar"
      }
    }
},
```

**Comment by phracek** [comment]
It makes sense to me, but personally, once we will over write all checks into internal documentation
then the ruleset file will be really big and obfuscated. 

What about to add a tag internal and add this tag into ruleset file.
I know that some checks will be duplicated.

Just brainstorming.

**Comment by TomasTomecek** [comment]
Fixed by #114.
